MVP.md

```diff
@@ -11,3 +12,27 @@ In all of our investigations to-date, we have discovered one protocol which has
 
 The best solution we have found is a provider known as [Unlock Protocol](https://unlock-protocol.com/) which offers ERC-721 tokens that are fully configurable and can be implemented on a variety of common L1 and L2 blockchains.  From a pure functional perspective
 
+</details>
+
+# Problem = Signal 2 Noise Ratio
+
+<details><summary>The tokenomics of LexDAO are simple but the interactions are subtle:</summary>
+1. mmmber - annual fixed subscriptionin fiat but payable in stablecoins;
```

Comment:
Can we say something like "member - annual fixed subscription pegged to a firm USD value but payable in multiple currencies both digital and fiat."